build.gradle

```diff
@@ -217,31 +224,31 @@ task ('cleanTest') {
 task renderConfigs(type: RenderConfigs) {
     local = false
     projectHome = projectDir
-    vault = System.getenv('VAULT_ADDR')
+    vaultAddr = System.getenv('VAULT_ADDR')
```

Comment:
This change seems unnecessary to me. In the current model, all configurations are in Circleci and rendered out via circleci.yml. If that is the route we are taking, then this task should be deleted, along with *all* of the code from `buildSrc`, which includes the `RenderConfigs` class.

Reply:
done